go.mod

```diff
@@ -2,46 +2,42 @@ module github.com/cloudnativelabs/kube-router
 
 require (
 	github.com/Microsoft/go-winio v0.4.14 // indirect
-	github.com/aws/aws-sdk-go v1.13.32
+	github.com/aws/aws-sdk-go v1.38.2
+	github.com/containerd/containerd v1.4.4 // indirect
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.9.1
 	github.com/coreos/go-iptables v0.5.0
 	github.com/dgryski/go-farm v0.0.0-20180109070241-2de33835d102 // indirect
 	github.com/docker/distribution v0.0.0-20170524205824-1e2f10eb6574 // indirect
-	github.com/docker/docker v0.0.0-20180414165938-5c2e6064a1c5
+	github.com/docker/docker v20.10.5+incompatible
 	github.com/docker/go-connections v0.3.0 // indirect
 	github.com/docker/go-units v0.3.3 // indirect
-	github.com/eapache/queue v1.1.0 // indirect
-	github.com/go-ini/ini v1.35.0 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/protobuf v1.4.3
-	github.com/gotestyourself/gotestyourself v2.2.0+incompatible // indirect
 	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce // indirect
-	github.com/jmespath/go-jmespath v0.0.0-20160202185014-0b12d6b521d8 // indirect
 	github.com/k-sone/critbitgo v1.4.0 // indirect
 	github.com/magiconair/properties v1.7.6 // indirect
 	github.com/moby/ipvs v1.0.1
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.3
+	github.com/moby/term v0.0.0-20201216013528-df9cb8a40635 // indirect
+	github.com/morikuni/aec v1.0.0 // indirect
+	github.com/onsi/ginkgo v1.15.2
+	github.com/onsi/gomega v1.11.0
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/opencontainers/image-spec v1.0.1 // indirect
-	github.com/osrg/gobgp v0.0.0-20200806055634-c6f0eba8f4d8
+	github.com/osrg/gobgp v0.0.0-20210101133947-496b372f7b8d
 	github.com/pelletier/go-toml v1.1.0 // indirect
-	github.com/prometheus/client_golang v0.9.4
-	github.com/smartystreets/goconvey v1.6.4 // indirect
+	github.com/prometheus/client_golang v1.10.0
 	github.com/spf13/cast v1.2.0 // indirect
 	github.com/spf13/jwalterweatherman v0.0.0-20180109140146-7c0cea34c8ec // indirect
 	github.com/spf13/pflag v1.0.5
 	github.com/vishvananda/netlink v1.1.1-0.20201029203352-d40f9887b852
-	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
-	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
-	google.golang.org/grpc v1.27.1
-	gopkg.in/ini.v1 v1.61.0 // indirect
-	gotest.tools v2.2.0+incompatible // indirect
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
-	k8s.io/cri-api v0.20.2
+	github.com/vishvananda/netns v0.0.0-20210104183010-2eb08e3e575f
+	golang.org/x/net v0.0.0-20210316092652-d523dce5a7f4
+	google.golang.org/grpc v1.36.0
+	k8s.io/api v0.20.5
+	k8s.io/apimachinery v0.20.5
+	k8s.io/client-go v0.20.5
+	k8s.io/cri-api v0.20.5
 )
 
 go 1.13
```

Comment:
@mrueg should this version here get bumped as well? To 1.15?

Reply:
I think we should still be fine with go 1.13, not we're not relying on anything that is more recent (this is a minimum version)